x-pack/plugins/security_solution/public/common/components/navigation/security_side_nav/security_side_nav.tsx

```diff
@@ -77,6 +77,7 @@ const useFormatSideNavItem = (): FormatSideNavItems => {
                     id: current.id,
                     label: current.title,
                     description: current.description,
+                    isBeta: current.isBeta ?? false,
```

Comment:
I think here we should trust what comes from the `NavLinkItem` type in this component.
If we want to set `false` to the `isBeta` as a default value, instead of leaving it optional, it would probably be better to centralize the assignment of the default value in the `formatNavLinkItems` function, so we don't have default value assignments in different parts of the code.